src/main/flight/navigation_rewrite_multicopter.c

```diff
@@ -483,11 +486,19 @@ bool isMulticopterLandingDetected(uint32_t * landingTimer)
     // from processRx() and rcCommand at that moment holds rc input, not adjusted values from NAV core)
     bool minimalThrust = rcCommandAdjustedThrottle < posControl.navConfig->mc_min_fly_throttle;
 
+    /*
+    Do not trust the sonar, it wants to kill you!
```

Comment:
It was not ok in the blackbox logs posted in the issue, have you had a chance to look at them? Urass sonar reads 4-6cm very stable while the sonar is out of range, and in stronnag's logs the reading was very noisy.
I guess a noisy sonar is easy to detect as unhealthy, but what about that stable out of range reading that looks exactly like it's sitting on the ground...?

I think it's a bad idea to only trust the sonar, but as a added safety, sure!